src/client/Microsoft.Identity.Client/PlatformsCommon/IClock.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.Identity.Client.PlatformsCommon
+{
+    internal interface IClock
```

Comment:
We already have an interface for this - https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/blob/9df321e3b71a9c673cda812710147fe48a907005/src/client/Microsoft.Identity.Client/Utils/ITimeService.cs#L8